roles/percona_xtradb_cluster/README.md

```diff
@@ -27,7 +27,7 @@ example below.
 ```yaml
 percona_xtradb_cluster_spec:
   backup:
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+    image: percona/percona-xtradb-cluster-operator:1.12.0-pxc5.7-backup
```

Comment:
Can we go with the same version as the cluster operator?
```suggestion
    image: percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup
```

Reply:
I don't see a problem in that. So probably this is ok.

Reply:
yeap, it should not include any breakage, just for version matching